### ISSUE TITLE
Early stopping

### ISSUE THREAD

**Issue by gngdb** [Neuroglycerin/neukrill-net-work#55]
Need a model that will quickly find a local optimum so we can change hyper-parameters and figure out what's holding our model back. Suspect a good way to do this is to use the same architecture we've been working with already (with extra convolutional layer) and turn down dropout in the final layers and reducing the rate of decay of the learning rate.
